tests/routes/v1/test_plates.py

```diff
@@ -17,58 +17,197 @@
 NEW_PLATE_ENDPOINTS = [prefix + NEW_PLATE_ENDPOINT for prefix in ENDPOINT_PREFIXES]
 GET_PLATES_ENDPOINTS = [prefix + GET_PLATES_ENDPOINT for prefix in ENDPOINT_PREFIXES]
 
-FILTERED_PLATE_TYPES: List[Optional[str]] = [None, SS_PLATE_TYPE_DEFAULT]
-UNFILTERED_PLATE_TYPES: List[Optional[str]] = ["unfiltered"]
+FIT_TO_PICK_PLATE_TYPES: List[Optional[str]] = [None, SS_PLATE_TYPE_DEFAULT, "fit_to_pick_new_samples_only"]
```

Comment:
The checks during CI always ensure that the linting has been done project wide.  There is a configuration for the linting in this project to limit line length to 120 characters.  It might seem a lot, but we don't work on 80 character monitors these days, so I am inclined to agree with the configuration.  120 characters is not hard to parse on a modern display.